src/components/button/LogoutButton.jsx

```diff
@@ -11,8 +11,10 @@ const LogoutButton = () => {
   return (
     <IonButton
       onClick={() => {
-        localStorage.removeItem("access_token");
-        localStorage.removeItem("refresh_token");
+        localStorage.removeItem("accessToken");
+        localStorage.removeItem("refreshToken");
+        localStorage.removeItem("userId");
+        localStorage.removeItem("isGuest");
```

Comment:
For guest users, I think we can have a different Toolbar, probably only need Home, Discover, and Profile (to sign up or sign in)